Workbooks/UpdateCompliance/UpdateComplianceHub/Update Compliance Hub.workbook

```diff
@@ -0,0 +1,1660 @@
+                                },
+                                "numberFormat": {
+                                  "unit": 0,
+                                  "options": {
+                                    "style": "decimal",
+                                    "useGrouping": true
+                                  }
+                                }
+                              },
+                              {
+                                "columnMatch": "DeploymentId",
+                                "formatter": 18,
+                                "formatOptions": {
+                                  "thresholdsOptions": "icons",
+                                  "thresholdsGrid": [
+                                    {
+                                      "operator": "is Empty",
+                                      "representation": "Blank",
+                                      "text": "N/A"
+                                    },
+                                    {
+                                      "operator": "Default",
+                                      "thresholdValue": null,
+                                      "representation": "Blank",
+                                      "text": "{0}{1}"
+                                    }
+                                  ]
+                                }
+                              },
+                              {
+                                "columnMatch": "KBNumberLink",
+                                "formatter": 1,
+                                "formatOptions": {
+                                  "linkColumn": "KBLinkText",
+                                  "linkTarget": "Url"
+                                }
+                              },
+                              {
+                                "columnMatch": "KBLinkText",
+                                "formatter": 5,
+                                "formatOptions": {
+                                  "linkTarget": "Url",
+                                  "linkIsContextBlade": false
+                                }
+                              },
+                              {
+                                "columnMatch": "TotalDevices",
+                                "formatter": 1,
+                                "formatOptions": {
+                                  "linkTarget": "WorkbookTemplate",
+                                  "linkIsContextBlade": false,
+                                  "workbookContext": {
+                                    "componentIdSource": "workbook",
+                                    "resourceIdsSource": "workbook",
+                                    "templateIdSource": "static",
+                                    "templateId": "community-workbooks/UpdateCompliance/Quality Updates/resourceGroups/EdpTest2/providers/microsoft.insights/workbooks/d931a806-fb13-41e3-86fa-dbd631340901",
```

Comment:
remove resourcegroups part and double check these links are valid from your storage account